swagger/v1/openapi.json

```diff
@@ -72,6 +72,17 @@
                 }
               ]
             },
+            "items": {
+              "enum": [
+                "title",
+                "version",
+                "title:asc",
+                "title:desc",
+                "version:asc",
+                "version:desc"
+              ],
+              "type": "string"
+            },
```

Comment:
Hi ! this section doesn't seem to belong to the `parameter` level but to the `schema` one, better something like 

```
        - name: sort_by
          in: query
          required: false
          description: >-
            An array of fields with an optional direction (:asc or :desc) to
            sort the results.
          schema:
            items:
              enum:
                - title
                - version
                - title:asc
                - title:desc
                - version:asc
                - version:desc
            type: string
```

Also check `type` should not be specified neither at the same level, but under schema. 

The same revision should be applied to the rest of the items in this PR.